packages/camera/camera_android_camerax/README.md

```diff
@@ -49,6 +49,15 @@ Any specified `ResolutionPreset` wll go unused in favor of CameraX defaults and
 
 `setZoomLevel` is unimplemented.
 
+## Known Issue
```

Comment:
Nit: Known Issues

(Since we may have others in the future, and will probably forget to change it if we do.)